ui/app/src/components/BrowseFilesDialog/BrowseFilesDialogContainer.tsx

```diff
@@ -123,7 +137,7 @@ export function BrowseFilesDialogContainer(props: BrowseFilesDialogContainerProp
 
   const onCloseButtonClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => onClose(e, null);
 
-  return (
+  return currentPathExists ? (
```

Comment:
currentPathExists is initialized in true, so inner components skeletons will render while checking for existence.

Reply:
We discussed this. We don't want the inner requests triggered.